client/src/__tests__/RelatedItems.test.js

```diff
@@ -28,6 +30,20 @@ describe('RelatedProducts', () => {
     const heading = screen.getByText('YOUR OUTFITS');
     expect(heading).toBeInTheDocument();
   });
+
+  // test('should invoke setRelatedIdList function ', async () => {
```

Comment:
I think you want to check that your getRelatedList function is resolving the axios request correctly. If that happens, setRelatedIdList should be called with the data you get back. Maybe refactor this test to check the value of relatedIdList before and after the function is called to see the change